data-files/samples/trials.Experiment.Any

```diff
@@ -0,0 +1,142 @@
+{ 
+    description = "trials"; 
+    frameRate = 120; 
+    showHUD = true;
+    renderWeaponStatus = true;
+    cooldownMode = "ring";
+    cooldownColor = Color4(0.7,1.0,0.9,0.75);
+            
+    weapon = { 
+        id = "2hit"; 
+        firePeriod = 0.3;
+        damagePerSecond = 2;
+        autoFire = false;
```

Comment:
`autoFire` defaults to `false` but we are setting it here, not an issue but I might remove it from below since its repeated quite a bit.